GenderPayGap.WebUI/GenderPayGap.WebUI.csproj

```diff
@@ -108,6 +108,9 @@
     <ProjectReference Include="..\GovUkDesignSystem\GovUkDesignSystem.csproj" />
   </ItemGroup>
   <ItemGroup>
+    <None Update="AzureAttachAutoscalingPolicy.sh">

```

Comment:
Don't we need to include `AzureDeployIPDeny.sh` as well?

Reply:
It's not included in the WebUI, there's a separate export step that handles the IP deny stuff that will get it

Reply:
I see, thanks!